### PR TITLE
APR-1477 RoATP FHA functional & design update

### PR DESCRIPTION
**Pull request by martin-hornsby**

### CODE REVIEW

File: src/SFA.DAS.ApplyService.Domain/Apply/RoaptFinancialApplicationsStatusCounts.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace SFA.DAS.ApplyService.Domain.Apply
+{
+    public class RoaptFinancialApplicationsStatusCounts
```
Comment (David-B-Read) [inline src/SFA.DAS.ApplyService.Domain/Apply/RoaptFinancialApplicationsStatusCounts.cs]: Should be RoatpFinancialApplicationsStatusCounts

File: src/SFA.DAS.ApplyService.Domain/Apply/RoaptFinancialApplicationsStatusCounts.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    {
+        public int ApplicationsOpen { get; set; }
+        public int ApplicationsWithClairification { get; set; }
```
Comment (David-B-Read) [inline src/SFA.DAS.ApplyService.Domain/Apply/RoaptFinancialApplicationsStatusCounts.cs]: Typo in property name

File: src/SFA.DAS.ApplyService.Data/ApplyRepository.cs [hunk 3]
```diff
@@ -478,6 +478,21 @@ FROM OPENJSON(i.Sequences)
             }
         }
 
+        public async Task<RoaptFinancialApplicationsStatusCounts> GetFinancialApplicationsStatusCounts()
+        {
+            // Note: For now it is easier to run all three queries. It may make sense to do something similar to that done with EPAO
```
Comment (David-B-Read) [inline src/SFA.DAS.ApplyService.Data/ApplyRepository.cs]: Should be ok for Private Beta, but we might want to look at just getting the counts once more applications start to come through

File: src/SFA.DAS.ApplyService.Domain/Apply/RoaptFinancialApplicationsStatusCounts.cs (hunk 1 above)
Comment (martin-hornsby) [reply src/SFA.DAS.ApplyService.Domain/Apply/RoaptFinancialApplicationsStatusCounts.cs]: corrected

File: src/SFA.DAS.ApplyService.Domain/Apply/RoaptFinancialApplicationsStatusCounts.cs (hunk 2 above)
Comment (martin-hornsby) [reply src/SFA.DAS.ApplyService.Domain/Apply/RoaptFinancialApplicationsStatusCounts.cs]: corrected

**Pull request merged**